src/views/SpaceSettings.vue

```diff
@@ -48,19 +58,25 @@ const modalVotingTypeOpen = ref(false);
 const modalPluginsOpen = ref(false);
 const modalValidationOpen = ref(false);
 const loaded = ref(false);
+const settingENSRecord = ref(false);
 const uploadLoading = ref(false);
 const showErrors = ref(false);
 const delayUnit = ref('h');
 const periodUnit = ref('h');
+const loadingEnsDomains = ref(false);
+
 const form = ref({
   strategies: [],
   categories: [],
   plugins: {},
   filters: {},
+  admins: [],
```

Comment:
I was wondering but I understand this line now. :D

Running into the same:
![image](https://user-images.githubusercontent.com/6792578/153495792-801fb9a9-b2bb-470b-a4dd-3a523d609d0a.png)

Didn't look into it very closely yet but maybe we can prepare the form right from the validation schema.

cc @samuveth